core/chains/evm/config/config.go

```diff
@@ -41,11 +43,11 @@ type ChainScopedOnlyConfig interface {
 	EthTxReaperThreshold() time.Duration
 	EthTxResendAfterThreshold() time.Duration
 	EvmFinalityDepth() uint32
-	EvmGasBumpPercent() uint16
+	EvmGasBumpPercent() uint32
```

Comment:
Why does this need such a large range? I thought we were trying to go the other way: tighten these as much as possible to get up-front validation against absurd values.

Reply:
Yeah, not sure why I changed this. I can't imagine you'd want to bump by more than 65000%.

Will put it back.